src/Symfony/Bundle/SecurityBundle/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 4.4.0
 -----
 
+ * Added `migrating_from` option to encoders configuration.
```

Comment:
Perhaps it should be named `migrate_from` (verb) for consistency with the other options, e.g. `ignore_case`.

Reply:
done!